src/Microsoft.Health.Dicom.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -11,14 +11,14 @@ public static class KnownRoutes
         private const string SeriesRouteSegment = "series";
         private const string InstancesRouteSegment = "instances";
         private const string MetadataSegment = "metadata";
-        private const string CustomTagsRouteSegment = "tags";
```

Comment:
Looks like all URI are lower case, could you do the same?

I'm thinking what's the best URI should we use here? _tags_ looks too generic, while _extendedquerytags_ feels a bit long, maybe got some input from team? But I don't think it's a blocker,  could be renamed anytime.